hasura.planx.uk/metadata/tables.yaml

```diff
@@ -1060,7 +1060,8 @@
           - team_id
           - updated_at
           - version
-        filter:
+        filter: {}
+        check:
```

Comment:
These `filter` and `check` blocks correspond to pre-update and post-update checks respectively in the Hasura console.